uber/model_checks.py

```diff
@@ -70,6 +70,9 @@ def group_paid(group):
         return "What you entered for Amount Paid ({}) isn't even a number".format(group.amount_paid)
 
 
+def _valid_donation(s):
+        return (s.isnumeric() and val(s) > -1 and val(s) < 1000000 and (val(s) = int(val(s))))
```

Comment:
I think you mean ``int(s)`` rather than ``val(s)``, since the latter isn't actually a thing, unless I'm missing something.